modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/viewmodel/PlayerViewModel.kt

```diff
@@ -395,6 +396,28 @@ class PlayerViewModel @Inject constructor(
         }
     }
 
+    fun toggleNotifications(context: Context) {
+        Toast.makeText(context, LR.string.player_effects_custom_effects_applied, Toast.LENGTH_SHORT).show()
+    }
+
+    fun updateOverrideGlobalEffects(override: Boolean) {
```

Comment:
Here, `override` looks like a pre-defined naming convention. Can we use self-explanatory name?